acceptance/localcluster/localcluster.go

```diff
@@ -410,22 +417,18 @@ func (l *Cluster) Start() {
 	l.mu.Lock()
 	defer l.mu.Unlock()
 
+	l.monitorStopper = make(chan struct{})
+
 	l.runDockerSpy()
 	l.initCluster()
 	l.createCACert()
 	l.createNodeCerts()
 	l.createClientCerts()
-
-	l.monitorStopper = make(chan struct{})
```

Comment:
not sure I understand. I think this is the definite order: create the channel early (because once `Start()` is invoked, it's reasonable to close the channel so it shouldn't be `nil`) and starting the goroutine after all event-dispatching calls (so that no unforeseen events reach it).